docs/guides/latest/platforms/aks.md

```diff
@@ -0,0 +1,643 @@
+$ kubectl apply -f ./docs/examples/plateorms/aks/restoresession.yaml
+restoresession.stash.appscode.com/deployment-restore created
+```
+
+Once, you have created the `RestoreSession` crd, Stash will inject `init-container` to `stash-recovered` Deployment. The Deployment will restart and the `init-container` will recover on start-up.
+
+**Verify Init-Container:**
+
+Wait until the `init-container` has been injected to the `stash-recovered` Deployment. Let’s describe the Deployment to verify that the `init-container` has been injected successfully.
+
+```yaml
+$ kubectl describe deployment -n demo stash-recovered
+Name:                   stash-recovered
+Namespace:              demo
+CreationTimestamp:      Thu, 18 Jul 2019 11:59:41 +0600
+Labels:                 app=stash-demo
+Selector:               app=stash-demo
+Replicas:               3 desired | 3 updated | 3 total | 3 available | 0 unavailable
+Pod Template:
+  Labels:       app=stash-demo
+  Annotations:  stash.appscode.com/last-applied-restoresession-hash: 15483804576325149444
+  Init Containers:
+   stash-init:
+    Image:      suaas21/stash:volumeTemp_linux_amd64
+    Port:       <none>
+    Host Port:  <none>
+    Args:
+      restore
+      --restore-session=deployment-restore
+      --secret-dir=/etc/stash/repository/secret
+      --enable-cache=true
+      --max-connections=0
+      --metrics-enabled=true
+      --pushgateway-url=http://stash-operator.kube-system.svc:56789
+      --enable-status-subresource=true
+      --use-kubeapiserver-fqdn-for-aks=true
+      --enable-analytics=true
+      --logtostderr=true
+      --alsologtostderr=false
+      --v=3
+      --stderrthreshold=0
+    Environment:
+      NODE_NAME:   (v1:spec.nodeName)
+      POD_NAME:    (v1:metadata.name)
+    Mounts:
+      /etc/stash/repository/secret from stash-secret-volume (rw)
+      /source/data from restore-data (rw)
+      /tmp from tmp-dir (rw)
+  Containers:
+   busybox:
+    Image:      busybox
+    Port:       <none>
+    Host Port:  <none>
+    Args:
+      sleep
+      3600
+    Environment:  <none>
+    Mounts:
+      /restore/data from restore-data (rw)
+  Volumes:
+   restore-data:
+    Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
+    ClaimName:  restore-pvc
+    ReadOnly:   false
+   tmp-dir:
+    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
+    Medium:
+    SizeLimit:  <unset>
+   stash-podinfo:
+    Type:  DownwardAPI (a volume populated by information about the pod)
+    Items:
+      metadata.labels -> labels
+   stash-secret-volume:
+    Type:        Secret (a volume populated by a Secret)
+    SecretName:  azure-secret
+    Optional:    false
+  ...
+```
+
+Notice the `Init-Containers` section. We can see that the init-container `stash-init` has been injected which is running `restore` command.
+
+**Wait for RestoreSession to Succeeded:**
+
+Run the following command to watch RestoreSession phase,
```

Comment:
Now, wait for the restore process to complete. You can watch the RestoreSession phase using the following command,